express/index.cjs

```diff
@@ -219,11 +388,15 @@ async function checkUser(req, user, email, table, company) {
   }
 }
 
-/* OR (email = "' +
-        email +
-        '" AND company = "' +
-        company + */
+function logger(writeOut, newTime, address, source, user) {
+  writer.write(`${setTimestamp(newTime)} | status: ${writeOut.status != null ? writeOut.status : 500} | source: ${source} | error: ${writeOut.error} | reason: ${writeOut.reason} | user: ${user}@${address}\n`);
+}
+
+function errLogger(errOut, newTime, address, source, user) {
+  writer.write(`${setTimestamp(newTime)} | status: ${errOut.status != null ? errOut.status : 500} | source: ${source} | error: ${errOut.error} | reason: ${errOut.reason} | user: ${user}@${address}\n`);
+}
 
+// Human readable timestamp for log
 function setTimestamp(timeUpdate) {
```

Comment:
We should probably move these helper functions into a separate `utils` or `helpers` file to keep the index file from getting too bloated.